packages/components/src/ui/color-picker/hex-input.tsx

```diff
@@ -38,7 +38,12 @@ export const HexInput = ( { color, onChange, enableAlpha }: HexInputProps ) => {
 		<InputControl
 			__unstableInputWidth="8em"
```

Comment:
I know this is not caused by this PR, but I wonder if we could refactor away from the `__unstableInputWidth` property in this case as well?

(cc @sarayourfriend in case there's a specific reason why we're using this prop)

Reply:
I don't have any insight into this prop, sorry!

Reply:
That's fine! Thank you for replying anyway :)

Reply:
I removed the usage of __unstableInputWidth 👍